### DISCUSSION TITLE
如何在抖音内获取手机号

### DISCUSSION THREAD

**Discussion by qingweiSun** [NervJS/taro#13096] [Q & A] [answered]
### 这个特性解决了什么问题？
在抖音内获取手机号
抖音文档：https://developer.open-douyin.com/docs/resource/zh-CN/mini-app/develop/guide/open-capabilities/acquire-phone-number-acquire/

### 这个 API 长什么样？
有类似Taro.getPhone的方法

**Comment by joket1999** [comment, marked as answer]
看抖音的文档，其实跟微信一样。
```jsx
<Button open-type="getPhoneNumber" onGetPhoneNumber={this.getPhoneNumber}></Button>
```
将在Button组件的onGetPhoneNumber事件中获取到vi和encryptedData传给后端，在后端用session_key解密。
